src/clear.js

```diff
@@ -12,4 +12,8 @@ const clearAll = () => {
       deleteTask(e);
     }
   });
+  let count = 0;
+  const data = Array.from(localData).filter((e) => e.completed === false);
+  data.map((e) => e.index = count += 1);
+  setStorage(data);
 };
```

Comment:
- [ ] The user should get the correct list values after the page reloads. Please make sure when you reload the page the checkboxes next to the tasks are checked if the tasks are completed.